guide/src/main/asciidoc/resources-collection.adoc

```diff
@@ -27,7 +27,7 @@ The representation of a collection MUST contain a list of links to child resourc
 * Each item object MAY be extended with some key business properties, needed for display in a master view.
 * In case the collection is empty, the `items` property MUST have an empty array as value.
 * The `title` attribute MAY be used to provide a human-readable description for an item, usable as display text for the link.
-* `total` is the reserved word for all the items in the collection result.
+* `total` is the reserved word for the number of items in the collection result set. It SHOULD be present when pagination is used or MAY be present otherwise.
```

Comment:
Maybe "MAY/OPTIONAL" in all cases instead of "SHOULD''.

It could be technically difficult to always provide a total.

It's even advised against in the zalando guidelines:

https://opensource.zalando.com/restful-api-guidelines/#161

> Remark: You should avoid providing a total count unless there is a clear need to do so. Very often, there are significant system and performance implications when supporting full counts. Especially, if the data set grows and requests become complex queries and filters drive full scans. While this is an implementation detail relative to the API, it is important to consider the ability to support serving counts over the life of a service.

Reply:
following WG meeting: Jonas will adapt to make "total" always optional